README.md

```diff
@@ -40,7 +40,6 @@ See the file COPYING for more details.
 
 # The Build Process
 
-Refer to the file 'docs/src/code/building-linuxcnc.txt' for information
-about building and running the software, or look here:
-
-http://linuxcnc.org/docs/devel/html/code/building-linuxcnc.html
+Refer to the file 'docs/src/code/building-linuxcnc.adoc' for information
+about building and running the software, or look
+[here](https://linuxcnc.org/docs/devel/html/code/building-linuxcnc.html).
```

Comment:
Hmm I don't know if it is clever to link to the devel docs. At some point 2.9 will be stable and the link has to be changed again.

Reply:
I switched the link to GitHub, but that could also be out of sync. If relative links always work in markdown that might be better. Maybe it's preferred to drop the link.

Reply:
Yes, that makes also sense to link inside GitHub when coming from GitHub.